__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("Get /api/topics endpoint should respond with a array of topic objects, each haveing slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toBeInstanceOf(Array);
+
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
```

Comment:
Id like to see an expect statement for the length of the array. A forEach will pass on an empty array so we have to check the length